app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_like_counter
+
+  def update_like_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] It is considered good practice to declare callback methods as private. If left public, they can be called from outside of the model and violate the principle of object encapsulation. Kindly, make ```update_like_counter``` private.